### PR TITLE
Feature/hrt360 activity timeline

### PR DESCRIPTION
**Pull request by jasattilia**

### CODE REVIEW

File: lib/gateways/applications-api.ts [hunk 1]
```diff
@@ -179,3 +181,24 @@ export const getStats = async (): Promise<Array<Stat> | null> => {
...
+): Promise<ActivityHistoryPagedResult | null> => {
+  try {
+    const url = `${process.env.ACTIVITY_HISTORY_API}/activityhistory?targetId=${id}&pageSize=100`;
+
+    const cookies = cookie.parse(req.headers.cookie ?? '');
+    const parsedToken = cookies['hackneyToken'];
+
+    const { data } = await axios.get(url, {
+      headers: {
+        Authorization: 'Bearer ' + parsedToken,
```
Comment (tcmorris) [inline lib/gateways/applications-api.ts:197]: For the other Hackney APIs, we used x-api-key?

File: components/admin/application-history.tsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  };
+
+  const functionDelegate = activityText[historyItem.activityType];
+
+  if (functionDelegate) {
+    return functionDelegate(historyItem);
+  }
+}
+
+const getFormattedDate = (
```
Comment (tcmorris) [inline components/admin/application-history.tsx:73]: Common util, this probably already exists or should be shared.

File: lib/gateways/applications-api.ts (hunk 1 above)
Comment (jasattilia) [reply lib/gateways/applications-api.ts:197]: dunno, but this works.. i can test it

File: lib/gateways/applications-api.ts [hunk 3]
```diff
@@ -179,3 +181,24 @@ export const getStats = async (): Promise<Array<Stat> | null> => {
     return null;
   }
 };
+
+export const getApplicationHistory = async (
+  id: string,
+  req: any
+): Promise<ActivityHistoryPagedResult | null> => {
+  try {
+    const url = `${process.env.ACTIVITY_HISTORY_API}/activityhistory?targetId=${id}&pageSize=100`;
```
[the remark this answers is not in the document]
Comment (jasattilia) [reply lib/gateways/applications-api.ts:190]: done

**Pull request merged**